jmclient/jmclient/taker.py

```diff
@@ -293,7 +292,8 @@ def prepare_my_bitcoin_data(self):
             #to make failures unlikely while keeping the occurence of failure to
             #find sufficient utxos extremely rare. Indeed, a doubling of 'normal'
             #txfee indicates undesirable behaviour on maker side anyway.
-            self.total_txfee = 2 * self.txfee_default * self.n_counterparties
+            self.total_txfee = estimate_tx_fee(3, 2,
+                txtype=self.wallet.get_txtype()) * self.n_counterparties
```

Comment:
Tests are failing on this because taker doesn't have an attribute wallet anymore, I think change it to `wallet_service`.

Reply:
How odd. Wonder why my manual tests did not catch this. No time right now, hope to get this fixed later today.

Reply:
Fixed now.

Retrospect why I didnt catch this before: I've only seen one of the self.wallet.get_txtype() instances and replaced & tested for that (those in the big "else" block in prepare_my_bitcoin_data()). I missed the first one, therefore also didnt test that branch.

+1 for automated testing, it worked great in this case